spyderlib/spyder.py

```diff
@@ -1268,6 +1274,11 @@ def post_visible_setup(self):
         self.extconsole.setMinimumHeight(0)
 
         if not self.light:
+            # Update toolbar visibility status
+            self.load_last_visible_toolbars()
+            toolbars_visible = CONF.get('main', 'toolbars_visible')

```

Comment:
This should be called `show_toolbar_o`. It's a Spyder custom to call option variables that way :-)